api/src/main/java/com/google/appengine/api/datastore/DataTypeTranslator.java

```diff
@@ -258,11 +258,12 @@ private static void addListPropertyToPb(
         // If the value is indexed it appears in queries, but distinction between
         // null and empty list is lost.
       }
+      property.setValue(PropertyValue.getDefaultInstance());
```

Comment:
Might want help from datastore team there. I am worried about the comment "// Backward compatible behavior", the logic could break moving to proto2... Also, I do not fully understand the logic of setting a default value, while the method arguments above take a collections of values.

Reply:
Since value is one of the required fields, not setting it is causing build() to fail